src/llamafactory/data/parser.py

```diff
@@ -97,20 +97,29 @@ def get_dataset_list(dataset_names: Optional[Sequence[str]], dataset_dir: str) -
 
     dataset_list: List["DatasetAttr"] = []
     for name in dataset_names:
-        if dataset_info is None:  # dataset_dir is ONLINE
-            load_from = "ms_hub" if use_modelscope() else "hf_hub"
+        if dataset_info is None: # dataset_dir is ONLINE
+            load_from = None
+            if use_openmind():
+                load_from = "om_hub"
+            if use_modelscope():
+                load_from = "ms_hub"
+            if load_from is None:
+                load_from = "hf_hub"
```

Comment:
please use `if...elif...else` here

Reply:
Fixed, many thanks!

Reply:
Fixed in the latest commit, but this comment has not been marked as `Outdated` for it only shows the three previous commits.